recipes/core/models.py

```diff
@@ -0,0 +1,24 @@
+"""All django models required for recipe app
+
+"""
+from django.db import models
+
+MAX_NAME_LENGTH = 256
+
+
+class Recipe(models.Model):
+    """Model representing a recipe
+
+    """
+    name = models.CharField(max_length=MAX_NAME_LENGTH)
```

Comment:
`TextField` is best since we are using Posgres (no performance penalty really) see https://stackoverflow.com/a/7354680/9839950

Reply:
Interesting! Thanks!